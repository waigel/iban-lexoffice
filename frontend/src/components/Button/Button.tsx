import React from 'react';
import {classNames} from '../../utils/classNames';
export type ButtonProps = React.HtmlHTMLAttributes<HTMLButtonElement>;

export const Button = (props: ButtonProps) => {
  return (
    <button
      {...props}
      className={classNames(
        props.className,
        'bg-white px-4 py-2 rounded-3xl border text-black hover:scale-110 font-omessemibold'
      )}
      type="submit"
    />
  );
};
