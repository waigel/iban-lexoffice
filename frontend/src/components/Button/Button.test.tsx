import React from 'react';
import TestRenderer from 'react-test-renderer';
import {Button} from './Button';

it('should test the Button component is rendered correctly', () => {
  const tree = TestRenderer.create(<Button />).toJSON();
  expect(tree).toMatchSnapshot();
});
