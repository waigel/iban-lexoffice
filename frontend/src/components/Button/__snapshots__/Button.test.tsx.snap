// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test the Button component is rendered correctly 1`] = `
<button
  className="bg-white px-4 py-2 rounded-3xl border text-black hover:scale-110 font-omessemibold"
  type="submit"
/>
`;
